source/common/config/filter_json.h

```diff
@@ -179,6 +180,16 @@ class FilterJson {
    */
   static void translateSquashConfig(const Json::Object& json_config,
                                     envoy::config::filter::http::squash::v2::Squash& proto_config);
+
+  /**
+   * Translate a v1 JSON HTTP IP Tagging config to a v2
```

Comment:
What's the standard for whether new features should support v1 API?  On some of my PRs recently, reviewers have advised omitting v1 support, but it's not entirely clear to me what the rules are.

Reply:
I don't think there is a standard for supporting V1. I decided to add V1 support since the schema was already there.

Reply:
@ccaraman if Lyft don't need it and others aren't asking, we should probably not add this to v1, since we're trying to gracefully turn it down.